innopoints/views/product.py

```diff
@@ -134,6 +135,14 @@ def create_product():
 
 class ProductDetailAPI(MethodView):
     """REST views for the Product model"""
+    def get(self, product_id):
+        """Get a single product."""
+        product = Product.query.get_or_404(product_id)
+        schema = ProductSchema(exclude=('description',
```

Comment:
why excluding the description?

Reply:
Right, I thought that the listing already gave all the info so I just copy-pasted. Will fix